src/components/Fortune/Fortune.component.jsx

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
 
-import { useFortune } from '../../utils/hooks/useFortune';
+// import { useFortune } from '../../utils/hooks/useFortune';
```

Comment:
Do not keep commented out code in your commits, you've got your Git history to have it back if needed.